lib/Admin.php

```diff
@@ -6,13 +6,25 @@ class Admin {
 
 	public static function init() {
 		$filter = add_filter('plugin_row_meta', array(__CLASS__, 'meta_links'), 10, 2);
+		$action = add_action('after_plugin_row_timber-library/timber.php', array(__CLASS__, 'show_deprecation_warning'), 10, 3);
```

Comment:
Yes, this is exactly what I meant. This catches the attention of people who are responsible for managing the plugins without being too intrusive.